nautobot/extras/tests/test_api.py

```diff
@@ -559,37 +561,80 @@ def test_run_job_object_var(self):
         data = {
             "data": job_data,
             "commit": True,
+            "schedule": {
+                "name": "test",
+                "interval": "future",
+                "start_time": str(datetime.now() + timedelta(minutes=1)),
+            },
```

Comment:
Should this be a new test case, rather than modifying the `test_run_job_object_var` test case? At a glance it seems like we're losing test coverage of the case of running a job immediately with the given `job_data`.

Reply:
In a way that’s true, sadly there is no way to get to the variables of a job that is scheduled immediately if no worker is running (since it gets queued immediately rather than a model being generated). This is a halfway solution: it means that we can test the object behavior without a Celery worker running (which, as of now, is as the CI and most devs operate).

Reply:
Got it, makes sense to me.